pkg/integrationteststatus/integration_test_status.go

```diff
@@ -33,6 +33,8 @@ const (
 	IntegrationTestStatusPending IntegrationTestStatus = iota + 1 // Pending
 	// Starting to handle an integration test scenario for a snapshot
 	IntegrationTestStatusInProgress // InProgress
+	// Integration PLR cancelled for this ITS and snapshot
+	IntegrationTestStatusCancelled
```

Comment:
```suggestion
	IntegrationTestStatusCancelled  // Cancelled
```

Reply:
this is a result of me not pushing my latest changes to the PR.